config/environments/development.rb

```diff
@@ -13,8 +13,8 @@
     port: 587,
     domain: "domain.of.sender.net",
     authentication: "plain",
-    user_name: "dave",
-    password: "secret",
+    user_name: "garvshah01",
+    password: "dbomppjiwwikftyh",
```

Comment:
we should NEVER push credentials on github.
Use env variables / rails secrets for this

Reply:
done